src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example.mdx

```diff
@@ -36,6 +36,7 @@ New Relic can instrument:
 * [Python](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/python)
 * [Go](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/go)
 * [Java](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/java)
+* [Ruby](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/sam/ruby)
```

Comment:
@fallwith @homelessbirds  This link seems to be broken (There is no example of SAM for Ruby). I landed here from this doc page: https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example/

Reply:
Hi @vicentllongo. I apologize for the confusion I have caused. There are multiple New Relic teams and PRs involved with the effort to support Ruby based AWS Lambda functions and I ended up submitting this documentation PR and asking for it to be merged too early.

The linked content for Ruby will appear once https://github.com/newrelic/newrelic-lambda-extension/pull/204 has been merged.

In the meantime, our [Ruby Lambda beta test instructions](https://gist.github.com/fallwith/ab8256f447dde65feb2b7e176813cca7) can be used.